mantidproject/mantid
# Add "Save Workspaces" tab to Reflectometry (Polref) GUI

As described in the related issue, this PR adds the "Save Workspaces" tab to the Reflectometry (Polref) interface as shown in the diagram, along with the required functionality. This new tab essentially has all the same functionality and design as a similar "Save Workspaces" window which may be found in Interfaces->ISIS Reflectometry->File->Save Workspaces.

**To test:**

Ensure all unit tests pass.

Ensure that setting each of the options available in the interface has an appropriate effect on the saved workspace(s).

Check that the layout and design of the interface largely matches that of the old interface window.

Fixes #17816 .

~~Blocked by #18056 .~~

Release notes have been updated.

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
